### PR TITLE
support for regular redis via redis-utils

### PR DESCRIPTION
**Pull request by reflog**
WARNING - This template is public, do not include any sensitive information

- [ ] Do the tests pass? Consistently?
- [ ] Did this change improve test coverage?
- [ ] Has it been reviewed/approved by relevant teammates?
- [ ] Can it be QA’d in staging or something like staging?
- [ ] Is the code in question being linted? If not, consider adding a linter step to CI. If yes, make sure the linter is happy.
- [ ] Do we know how we’re going to deploy this?
- [ ] Are we clear on what the support and maintenance impact is?
- [ ] Did you create new documentation? Is it accessible and in the right place?
- [ ] Has existing documentation been updated?
- [ ] Have you logged tickets for related technical debt with the label “techdebt”?

### CODE REVIEW

**Review by oxtoacart — APPROVED** [review]
LGTM.

File: cmd/flags.go [hunk 1]
```diff
@@ -50,7 +42,7 @@ func StartPprof() {
 		go func() {
 			log.Debugf("Starting pprof page at http://%s/debug/pprof", *PprofAddr)
 			if err := http.ListenAndServe(*PprofAddr, nil); err != nil {
-				log.Errorf("Unable to start PPROF HTTP interface: %v", err)
+				_ = log.Errorf("Unable to start PPROF HTTP interface: %v", err)
```
Comment (hwh33) [inline cmd/flags.go:45]: Is this to pacify some linting tool?  It's pretty typical to ignore errors for logging calls and this results in inconsistent style.  It's not a big issue though.

**Review by hwh33 — APPROVED** [review]
I left one minor comment.  Also needs a `go mod tidy`.  Looks good otherwise though!

File: cmd/flags.go (hunk 1 above)
Comment (reflog) [reply cmd/flags.go:45]: @hwh33 yep it was for linting. I personally don't like functions that ignore returned errors, explicit is better than implicit and all that jazz ;)

**Pull request merged**